tests/DecisionServiceTests/DecisionServiceTest.php

```diff
@@ -118,6 +118,55 @@ public function testGetVariationBucketsUserWhenExperimentIsRunning()
         );
     }
 
+    public function testGetBucketingIdBucketingIdAttributeNotString()
```

Comment:
Rename test methods such as
testGetBucketinIdWhenBucketingIdIsNotString . Similar for others.